src/special/webpack.js

```diff
@@ -135,6 +135,7 @@ function loadNextWebpackConfig(filepath) {
     plugins: [],
     module: { rules: [] },
     optimization: { splitChunks: { cacheGroups: {} } },
+    resolve: { alias: () => {} },
```

Comment:
From the usage you linked it seems like `alias` should be an object, and not a function, maybe I'm missing something?

Reply:
You're absolutely right, it should be an object. Fixed.